src/org/janelia/mipav/plugins/worm/untwisting/PlugInDialogVolumeRenderDualJanelia.java

```diff
@@ -2255,30 +2337,72 @@ private void init() {
 		dialogGUI.getContentPane().add(panel1, BorderLayout.NORTH);
 		dialogGUI.getContentPane().add(panel2, BorderLayout.SOUTH);
 
-		lutPanel = new JPanel(new GridLayout(2,1));
+		lutPanel = new JPanel(new GridLayout(2, 1));
 		lutTab = new JTabbedPane();
 		lutTab.addChangeListener(this);
 		lutPanel.add(lutTab, BorderLayout.CENTER);
-		opacityPanel = new JPanel( new BorderLayout() );
+		opacityPanel = new JPanel(new BorderLayout());
 		opacityTab = new JTabbedPane();
 		opacityTab.addChangeListener(this);
 		opacityPanel.add(opacityTab, BorderLayout.CENTER);
+		clipPanel = new JPanel(new BorderLayout());
+
+		// Create a panel for toggling between accurate and 3-color modes(not working
+		// currently)
+		accuratePanel = new JPanel(new BorderLayout());
+		accuratePanel.setPreferredSize(new Dimension(400, 400));
+		accuratePanel.setMaximumSize(new Dimension(400, 400));
```

Comment:
```suggestion
		accuratePanel.setPreferredSize(new Dimension(400, 400));
		accuratePanel.setMaximumSize(new Dimension(400, 400));
```
What happens if we remove this?

Reply:
the plot get larger vertically and horizontally

Reply:
- [ ] added //comment for the changes of this two line